src/verify-and-receive.ts

```diff
@@ -4,11 +4,12 @@ import type {
   EmitterWebhookEventWithStringPayloadAndSignature,
   State,
 } from "./types";
+import AggregateError from "aggregate-error";
```

Comment:
nit: move module imports to the top block below `import { verify } from "@octokit/webhooks-methods";`

we loosely follow the convention

- 1st block: `node:...` imports
- 2nd block: module imports
- 3rd block: relative imports